src/ODB2/odbapi2nc.py.in

```diff
@@ -20,7 +21,7 @@ import var_convert
 #      If not using these environments, then note ODB API must be built on the system with the
 #      ENABLE_PYTHON flag on or else the python module won't be there.
 odbPythonPath = os.getenv('ODBPYTHONPATH', os.environ['HOME'] + '/projects/odb/install/lib/python2.7/site-packages/odb')
-# print odbPythonPath
+# print(odbPythonPath)
```

Comment:
Is it necessary to have a default option under `HOME`?
What if someone prefers `/opt`.  
My suggestion is to fail if `ODBPYTHONPATH` is not found in the environment.
Please remove the commented print statement.

Reply:
This is a question for @svahl991. I believe that this was put in before we had ODB libraries installed in the containers. This would allow one to build the ODB libraries, install them in their home directory, and then access them. I would guess that this is not needed anymore.

I would vote for removing the odbPythonPath code and the commented print statements if those changes are acceptable.

Thanks!

Reply:
@srherbener is correct. Before the containers had ODB API libraries I used this code to tell the script where to find the ODB API Python libraries. (I chose the HOME directory as the default because it was the recommended installation location by the 'jedi-tools' repository at the time.)

I'm OK with removing the whole section now that the containers have ODB API. The script will then only run without modification inside the container, but it seems like that's sort of the plan for right now anyway.

I can make this change, or @srherbener can do it. I'm fine either way. Thanks for catching this!

Reply:
Removing this section of the code is good, we have a partial check for `HAVE_ODB_API`, although that does not guarantee the python module.

Also, this will only work on a container with ODB_API installed in the PYTHONPATH.
For HPC's or installations of ODB in locations other than PYTHONPATH, we will have to find where the python module for ODB_API is, and then add logic similar to `@SCRIPT_LIB_PATH@`.  There exists findODB_API.cmake in ecbuild, and it will have to be updated to locate the python module. This is do-able at a later point and can be re-visited when ODC is made available to the public.